source/features/reactions-avatars.tsx

```diff
@@ -20,13 +20,15 @@ interface Participant {
 }
 
 function getParticipants(button: HTMLButtonElement): Participant[] {
-	const currentUser = getUsername();
-	const users = button.getAttribute('aria-label')!
+	// Reaction buttons on releases and review comments have the list of people in their `title` attribute instead of `aria-label` #5150
+	const buttonTitle = button.getAttribute('title')!;
+	const users = (buttonTitle.length > 0 ? buttonTitle : button.getAttribute('aria-label')!)
```

Comment:
If it passes lint, this is equivalent
```suggestion
	const users = (buttonTitle || button.getAttribute('aria-label')!)
```